### ISSUE TITLE
Chosen Wounds

### ISSUE THREAD

**Issue by Xicoz** [WarEmu/WarBugs#15683]
my chosens wounds are all messed up (yes I did .getstats to verify my wounds and health are correctly displayed in UI)
![Bloodsinned_023](https://user-images.githubusercontent.com/44329952/74496698-1b1fee00-4ea1-11ea-9f67-052bdf64acba.jpg)

**Comment by Xicoz** [comment]
also I don't have any wounds debuffs

**Comment by SilverWF** [comment]
It's pretty obvious, that you have there Diminishing Rations 'buff'.
This debuff lowers HP of the zerging side - punishment for being overpopulated.

**Comment by Xicoz** [comment]
I don't believe the debuff states this. if it does my bad, if it doesnt, it should lol, debuffs should say what they do
